### ISSUE TITLE
Wrong zlib on Debian

### ISSUE THREAD

**Issue by keinstein** [Lazy-Newb-Pack/Lazy-Newb-Pack-Linux#51]
Hi, I get the following messages:

[distro_fixes] [INFO] Checking whether any distro specific fixes are required...
[distro_fixes] [INFO] OS: debian
[distro_fixes] [INFO] ARCH: x86_64
[distro_fixes] [INFO] VER: unstable
[distro_fixes] [INFO] DF_ARCH: 64-bit
[distro_fixes] [INFO] DF_BIN_LOCATION: /home/tobias/Spiele/LinuxLNP-0.44.04-rc1/df_linux/libs/Dwarf_Fortress
[distro_fixes] [WARN] 64bit 'Dwarf_Fortress' on unhandled 64bit OS detected. If you get 'missing file' errors, please open an issue on Github: https://github.com/Lazy-Newb-Pack/Lazy-Newb-Pack-Linux/issues.
[distro_fixes] [INFO] Attempting to use zlib at /usr/lib32/libz.so
[distro_fixes] [INFO] PRELOAD_LIB: /usr/lib32/libz.so
[distro_fixes] [INFO] LD_LIBRARY_PATH: 
[distro_fixes] [INFO] Done
ERROR: ld.so: object '/usr/lib32/libz.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.

I don't know what that means. Downloaded upgrades seem not to be installed automatically but that's before the console window opens. And some issues with Dwarf therapist occur: The memory checksum doesn't match (at least in the previous versions).

**Comment by lethosor** [comment]
Does DF still start? What DF version are you using?

**Comment by keinstein** [comment]
DF starts. I'm running the current LinuxLNP x86_64-linux (including DF 0.44.04). The upgrading problem exists for several versions, now. At some point LinuxLNP even failed to download the new LinuxLNP version.

**Comment by lethosor** [comment]
This issue is unrelated to whatever upgrade issues you're having. I'm not sure if anything in this repo affects automatic upgrades, actually, so you might have better luck in that pack's forum thread.

**Comment by carterscottm** [comment]
I'm not sure what's going on with your zlib problem, but if DF loads ok I think you should be fine.  It's most likely a path that needs to get added to the distro_fixes script for debian sid.

As for the automatic downloading of updated packs, the updated pack should download into the root of your pack, but it looks like PyLNP errors out when attempting to extract it.  I'll file an issue in the PyLNP repo for this.

**Comment by pookleblinky** [comment]
Probably not much help, but I got almost the same issue using this LNP version on : http://dffd.bay12games.com/file.php?id=13491  , except it said no 64-bit zlib found. It displayed only 

```[distro_fixes] [INFO] PRELOAD_LIB:```

with no attempt to use a preload as in your

```[distro_fixes] [INFO] Attempting to use zlib at /usr/lib32/libz.so```

DF loaded normally, but on loading a world the fps would drop to 8 (3) and would crash.

I fiddled around and found that setting the print mode to 2D let me load a game perfectly normally, at normal fps, across several graphics packs. Enabling TWBT consistently produced the crash.

This, combined with DFHack's console error that "multilevel is not a recognized command" across packages, makes me think that the problem lays in TWBT 6.34 and not LNP.

Using *this* package, I've not had any similar crashes but TWBT, while functional, lacks multilevel and renders GemSet particularly awfully.
